master/getting-started/kubernetes/installation/hosted/k8s-backend/index.md

```diff
@@ -25,14 +25,37 @@ You must have a cluster which meets the following requirements:
 
 ## Installation
 
-To install Calico, ensure you have a cluster which meets the above requirements and run the following command:
+To install Calico, ensure you have a cluster which meets the above requirements and run the following command.
```

Comment:
should probably be "run one of the following commands based on your Kubernetes version"